aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -211,6 +245,27 @@ module aptos_framework::aptos_governance {
         borrow_global<GovernanceConfig>(@aptos_framework).required_proposer_stake
     }
 
+    #[view]
+    public fun is_partial_voting_proposal(proposal_id: u64): bool acquires PartialVotingProposals {
+        if (!exists<PartialVotingProposals>(@aptos_framework)) {
+            return false
+        };
+        let partial_voting_proposals = borrow_global<PartialVotingProposals>(@aptos_framework);
+        table::contains(&partial_voting_proposals.proposals, proposal_id)
+    }
+
+    #[view]
+    /// Return voting power usage of a stake pool on a proposal. Only partial voting proposals could be queried.
+    public fun get_voting_power_usage(stake_pool: address, proposal_id: u64): u64 acquires PartialVotingProposals, VotingRecordsV2 {
```

Comment:
nit: voting power usage is not a good name for this. Also, should we just have a single view function that returns a tuple of vote (yes or no) and voting power associated with the vote?